public/index.html

```diff
@@ -26,6 +26,7 @@
       ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"<%= htmlWebpackPlugin.options.newRelicLicenseKey %>",applicationID:"<%= htmlWebpackPlugin.options.newRelicApplicationID %>",sa:1}
     </script>
 
+    <script type="text/javascript" src="https://flex.cybersource.com/cybersource/assets/microform/0.11/flex-microform.min.js"></script>
```

Comment:
Need to handle the case where this fails to load.

Reply:
i believe we can just handle the generic case of the variable being undefined (though i'm unsure of what the product requirements are in that case, just throw and log an error?)

Reply:
Yes, I think log the error and show the user the generic error message.